packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -133,7 +133,8 @@ export async function copyProjectTemplateAndReplace(
   let mainComponentName = newProjectName;
   const appJsonPath = await findUp('app.json', {cwd: destPath});
   if (appJsonPath) {
-    mainComponentName = JSON.parse(fs.readFileSync(appJsonPath, 'utf8')).name;
```

Comment:
Nit: Since parsing json is so common and fs-extra has special Json overloads `fs.parseJSon*<T>(...) : T`.Would it make sense to add these convenience functions.